sdk/test/opentelemetry/sdk/trace/export/console_span_exporter_test.rb

```diff
@@ -22,11 +22,11 @@
     $stdout = @original_stdout
   end
 
-  it 'accepts an Array of Spans as arg to #export and succeeds' do
+  it 'accepts an Array of SpanData as arg to #export and succeeds' do
     _(exporter.export(spans)).must_equal export::SUCCESS
   end
 
-  it 'accepts an Enumerable of Spans as arg to #export and succeeds' do
+  it 'accepts an Enumerable of SpanData as arg to #export and succeeds' do
```

Comment:
With the change at L13 (going from an array of `Span` to an array of `Object`) we're not really testing that the `console_span_exporter` accepts "an Enumerable of SpanData" (and same for the test at L25).

...although, I suppose one could make the case that we were _never_ testing for an array/enumerable of `SpanData` specifically, since we were actually sending an array of `Span`. 😆 

Do we have any desire to ensure that this exporter accepts specifically `SpanData`? If we do, we should probably update the test to actually assert that specifically (and maybe have the exporter fail otherwise). Otherwise we could rename the test to something like `it accepts an Enumerable as arg to #export and succeeds` since the exporter as-implemented really doesn't care?

FWIW part of the reason I noticed this is that we have an in-house `LogSpanExporter` for debugging that's less verbose (and I've been meaning to contribute it back, honestly). We inherit from the `ConsoleSpanExporter` and do actually assume that we're getting an array of `SpanData`, and so if that's an unsafe assumption I can/will update our exporter to double-check what it's getting first.

Reply:
It's a safe assumption (that you're getting an `Enumerable` of `SpanData`), but `ConsoleSpanExporter` doesn't actually care what it receives. If you do contribute your `LogSpanExporter` (ideally as improvements to the `ConsoleSpanExporter`), then a more strict test would be warranted.

Honestly, I was just in the file to fix some stuff actually related to the PR, and I noticed that this was obviously wrong. I'll fix the test to do what it says.

Reply:
Done